.gitlint

```diff
@@ -63,10 +63,21 @@ words=wip
 # regex=[^@]+@foo.com
 
 [ignore-by-title]
-# Allow empty body only when upgrading dependendies
+# Allow empty body only when upgrading dependencies
 regex=^⬆️.*$
 ignore=B6
 
+[ignore-by-title]
+# Allow empty body & wrong title pattern only when pyup upgrades dependencies
+regex=^Update (.*) from (.*) to (.*)$
+ignore=B6,UC1
+
+[ignore-by-title]
```

Comment:
good if it works to duplicate the rule. Otherwise, do both in the same regex...

Reply:
That's what I finally did. :cry: